fe/fe-core/src/main/java/com/starrocks/catalog/LocalTablet.java

```diff
@@ -573,6 +573,12 @@ public TabletStatus getColocateHealthStatus(long visibleVersion,
 
         // 2. check version completeness
         for (Replica replica : replicas) {
+            // do not check the replica that is not in the colocate backend set,
+            // this kind of replica should be drooped.
+            if (!backendsSet.contains(replica.getBackendId())) {
```

Comment:
Why the colocate tablet should be repaired after dropping the BE?

Reply:
The replica record in meta should be removed.

Reply:
The bug is: current sr will try to catch up the replica version of the dropped be, and it will never success.